### DISCUSSION TITLE
There's a strange problem that all buses' theta is constantly changing

### DISCUSSION THREAD

**Discussion by MuaYoi** [CURENT/andes#497] [Q&A]
I used the ieee39_full.xlsx to simulate and added an alter that the 'Ppf' of load 'PQ_1' is modified from 6 to 6.2 at time 5.0
From my own perspective, this change is not significant enough to destabilize the system, so the phase angle of the bus should fluctuate for a period of time before returning to stability.
My code is here:
`ps = andes.load(ieee39,setup=False)`
`ps.TDS.config.tf = 40`
`ps.TDS.config.tstep = 0.005`
`ps.PQ.config.p2p = 1.0`
`ps.PQ.config.p2i = 0`
`ps.PQ.config.p2z = 0`
`ps.PQ.config.q2q = 1.0`
`ps.PQ.config.q2i = 0`
`ps.PQ.config.q2z = 0`
`ps.add('Alter',dict(model='PQ',dev='PQ_1', src='Ppf',t=5.0,method='+',amount=0.2))`
`ps.setup()`
`ps.PFlow.run()`
`ps.TDS.run()`
The result confused me, because the voltage curve met my expections, like the second figure but the theta curve is abnormal like the first figure.
![fig1](https://github.com/CURENT/andes/assets/151932306/740436fd-6114-455f-a25a-d980725c2726)
![fig2](https://github.com/CURENT/andes/assets/151932306/698d4657-9caa-497a-8d76-f5c47c0d2506)

**Comment by MuaYoi** [comment]
Let me add a picture, which shows the output curves of all synchronous generators.
![fig3](https://github.com/CURENT/andes/assets/151932306/8a2884bd-a9ac-4d6c-a0a3-94f3c7324c53)

**Comment by jinningwang** [comment]
Check this, https://github.com/CURENT/andes/discussions/335
